src/EditorFeatures/Core.Wpf/InlineRename/UI/SmartRename/SmartRenameViewModel.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Threading;
+using Microsoft.VisualStudio.Text.Editor.SmartRename;
+
+namespace Microsoft.CodeAnalysis.InlineRename.UI.SmartRename
+{
+    internal class SmartRenameViewModel : INotifyPropertyChanged, IDisposable
```

Comment:
```suggestion
    internal sealed class SmartRenameViewModel : INotifyPropertyChanged, IDisposable
```

file scoped namespaces.